README.md

```diff
@@ -3,17 +3,11 @@
 This project is an extension for VS Code to provide the features of the [Checker Framework](https://checkerframework.org/),
 via the [Checker Framework Language Server](https://github.com/eisopux/checker-framework-languageserver).
 
-
```

Comment:
The file used two empty lines before headings, to make them easier to spot. Please remove this removal and a similar change below.

Reply:
Good morning Professor Dietl! Sorry for not resolving these requests sooner, as I had tested positive for COVID and could not get to them. I will get these fixed as soon as possible!